app/ducks/walletActions.js

```diff
@@ -187,7 +187,13 @@ export const fetchTransactions = () => async (dispatch, getState) => {
     payload: true,
   });
 
-  const txs = await walletClient.getTransactionHistory();
+  let txs = [];
+
+  try {
+    txs = await walletClient.getTransactionHistory();
+  } catch (e) {
+    //
```

Comment:
Don't we want to do something with this error? Tx history will be empty if an error occurs here.

Reply:
i was trying to handle the error when wallet name is not set, which should return an empty array, but now I realize this is better handled in `wallet/service` instead. 

because there are many PRs stacked on this one now, I will fix this issue in my next commit.

Reply:
e06c913